pages/careers/_open-positions.html

```diff
@@ -200,7 +204,7 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
     <p>
       If you're excited by the above:<br />
       <a
-        href="https://apply.workable.com/gruntwork/j/AD26F6ED87/apply/"
+        href="https://apply.workable.com/gruntwork/j/30C78A892A/"
```

Comment:
Should this have `/apply` otherwise the link says "Apply Now" but they are taken to a page which just mirrors all of this content and again has an "apply" button at the bottom.

Reply:
Oh, great catch! Fixed in: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/758/commits/8269f7eab476c035eacdcfde1da83625f15c9e33.